server/plugin/command.go

```diff
@@ -342,22 +397,80 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	if appErr = p.CreatePost(args.ChannelId, p.BotUserID, msg); appErr != nil {
+		return fmt.Sprintf("%s error creating the public post: %s", msg, appErr.Error())
+	}
+
+	return ""
 }
 
+func (p *Plugin) getSubscribedFeatures(channelID, owner, repo string) (Features, error) {
+	var previousFeatures Features
```

Comment:
Why is this named `previousFeatures`?

Reply:
These are Features or events of an subscription which are going to be overwritten due to creation of new subscription.